media-api/app/MediaApiComponents.scala

```diff
@@ -30,6 +32,7 @@ class MediaApiComponents(context: Context) extends GridComponents(context, new M
   val usageQuota = new UsageQuota(config, actorSystem.scheduler)
   usageQuota.quotaStore.update()
   usageQuota.scheduleUpdates()
+  applicationLifecycle.addStopHook(() => Future{usageQuota.stopUpdates()})
```

Comment:
This looks good. I'm assuming it's for cleanliness rather than it having any implications on authentication?